### DISCUSSION TITLE
Prerequisite install of MS Visual C++ fails but install doesn't realize it

### DISCUSSION THREAD

**Discussion by dbennett2843** [Romanitho/Winget-AutoUpdate#658] [Q&A]
I deployed WAU and found an app whitelisting tool I use was blocking the install of the Visual C++ prerequisite, but the installer attempted to complete. It ended up a failed partial install. Here's an image of what I saw
![2024-08-13_22-47-28](https://github.com/user-attachments/assets/172f5dfb-6aa1-4029-8f46-3c574ce5911b)

I'm posting this in the hopes that you can add a valid check on the install process and abort with an error if the prereq install fails

**Comment by Romanitho** [comment]
seems good to me
